internal/cli/atlas/atlas.go

```diff
@@ -86,6 +87,7 @@ func Builder() *cobra.Command {
 		customdns.Builder(),
 		cloudproviders.Builder(),
 		serverless.Builder(),
+		livemigration.Builder(),
```

Comment:
go test is failing because you added a command here, we test number of commands
probably easier to see the test failure on evergreen
https://spruce.mongodb.com/task/mongocli_master_code_health_unit_test_patch_ae1e135fe3b5f9f8500a5143169de2d0d7887ddb_612de81e850e61092f7ff48c_21_08_31_08_28_15/tests?execution=0&sortBy=STATUS&sortDir=ASC

Reply:
Doh, that error message totally confused me, thanks!